_minutes/2022-11-10-wecg.md

```diff
@@ -0,0 +1,182 @@
+ * [mukul] Will do.
+ * [rob] Edge is a fork of Chromium, so generally support from Chrome usually implies support in Edge.
+
+[PR 291](https://github.com/w3c/webextensions/pull/291): Document How we work
+
+ * [simeon] We are in a good position to merge these contribution guidelines.
+ * [rob] Should I update the PR with what we just agreed on the process first?
+ * [simeon] Let's merge it. Do multiple iterations of smaller updates.
+
+(follow-up from TPAC) Check in on status of gathering IDL/JSON for spec work
+
+ * [timothy] I am planning to add a directory to add IDL/JSON files for spec work.
+ * [tomislav] We are in the process of generating IDL files from JSON schemas. Depending on timing we could either commit the JSON or the WebIDL.
+ * [rob] I know that Chrome and Firefox have implementation details embedded in the JSON/IDL files that are not part of the actual public extension API. These should ideally not be included in the spec.
+ * [timothy] This is also the case in the Safari IDL. I am planning to remove this.
+ * [simeon] We could also start with the raw version and clean up in updates.
+
+[Issue 12](https://github.com/w3c/webextensions/issues/12): request: allow to retrieve a frameID from an &lt;iframe> element
+
+ * [simeon] We were all generally supportive of exposing this feature.
+ * [rob] Both Firefox and Safari have implemented this. Status of Chrome?
+ * [simeon] We have not followed up on this. We like the idea but have not prioritized the implementation. I will try to follow up to get an estimate on when to tackle it. Chrome has a fix-it backlog where engineers can occasionally tackle smaller issues.
+
+[Issue 188](https://github.com/w3c/webextensions/issues/188): Inconsistency: Permissions "menus"
+
+ * Check-in on support for replacing contextMenus with menus and (2) changing the API signature of menus.create / contextMenus.create
+ * [simeon] I have previously expressed that we do not intend to drop the contextMenus namespace. I will post a comment here.
+
+[Issue 282](https://github.com/w3c/webextensions/issues/282): Proposal: declaring background scripts in a neutral way
+
+ * Check-in on support for declaring bg scripts in a neutral way
+ * [simeon] Last night I updated the issue with Chrome's position. We have significant reservations about doing this due to the non-trivial differences between the execution environments. We are open to additional exploration, we are not opposed to a single manifests that works everywhere. We are interested in well-defined semantics. If a developer declares that an extension works in an Event page, a Service worker and a Background page, then it is not obvious whether that is really the case.
+ * [rob] The feature request already adds a key to opt in, isn't that a clear enough signal?
+ * [simeon] preferredEnvironment is not a clear enough signal. Maybe it's just a matter of naming.
```

Comment:
```suggestion
 * [simeon] No, we don't think `preferredEnvironment` is not a clear enough signal. That name implies deference rather than an explicit declaration. Maybe it's just a matter of naming.
```